src/webapp/azext_webapp/__init__.py

```diff
@@ -28,7 +28,7 @@ def load_command_table(self, _):
     def load_arguments(self, _):
         with self.argument_context('webapp new') as c:
             c.argument('name', options_list=['--name', '-n'], help='name of the new webapp')
-            c.argument('dryrun',
+            c.argument('dryrun', options_list=['--dryrun', '-dr'],
```

Comment:
The short option should only be a single character.
https://docs.python.org/3/library/argparse.html#option-value-syntax